requirements.txt

```diff
@@ -5,7 +5,7 @@ natsort
 numpy==1.18.1
 opencv-python==3.4.2.17
 openpyxl>=2.6.1
-pandas==1.0.2
+pandas>=1.0.2
```

Comment:
I haven't tried running pysero with version higher than 1.0.2. I have no problem with upgrading pandas version if it doesn't break things...